zcash_primitives/CHANGELOG.md

```diff
@@ -22,7 +22,10 @@ workspace.
 - Removed deprecated `zcash_primitives::transaction::sighash::SIGHASH_ANYONECANPAY`, use `zcash_transparent::sighash::SIGHASH_ANYONECANPAY` instead.
 - Removed deprecated `zcash_primitives::transaction::sighash::SighashType`, use `zcash_transparent::sighash::SighashType` instead.
 
-- Removed deprecated module `zcash_primitives::transaction::components::amount::testing`, use `zcash_protocol::value::testing` instead.
+- Removed deprecated module `zcash_primitives::transaction::components::amount::testing`:
+  - `arb_positive_amount`, use `zcash_protocol::value::testing::arb_positive_zat_balance` instead.
+  - `arb_amount`, use `zcash_protocol::value::testing::arb_zat_balance`
+  - `arb_nonnegative_amount`, use `zcash_protocol::value::testing::arb_zatoshis` 
```

Comment:
```suggestion
- Removed deprecated module `zcash_primitives::transaction::components::amount::testing`,
  use `zcash_protocol::value::testing` instead with the following renames:
  - `arb_positive_amount` to `zcash_protocol::value::testing::arb_positive_zat_balance`
  - `arb_amount` to `zcash_protocol::value::testing::arb_zat_balance`
  - `arb_nonnegative_amount` to `zcash_protocol::value::testing::arb_zatoshis`.
```
Also consider squashing this commit into the first one if you're going to be rebasing.